bessctl/conf/testing/run_module_tests.bess

```diff
@@ -155,15 +155,14 @@ def run_tests():
                     except socket.timeout:
                         return_pkt = None
                     if(not ((return_pkt is None and output_pkt is None) or
-                            (return_pkt == scapy.Ether(output_pkt)))):
-                        sys.stderr.write("Test failed!")
+                            (str(return_pkt) == str(output_pkt)))):
+                        sys.stderr.write("Test failed!\n")
                         sys.stderr.write("Input packet: %s" %
-                                         pkt_str(scapy.Ether(input_pkt)))
+                                         pkt_str(input_pkt) + "\n")
```

Comment:
Perhaps both packet data in hex and `pkt.summary()`should be printed out? Hex data alone may be hard for users to reason about "semantic: differences...